tvb/simulator/plot/tools.py

```diff
@@ -520,8 +681,204 @@ def channel_formatter(x, pos=None):
 
     return fig
 
+def plot_with_weights(weights):
+    """
+    Connectivity normalization
+    With significant variability between methods of obtaining structural connectivity,
+    it is often useful to normalize, in one sense or another, the connectivity when
+    comparing, for example, across subjects, or standardizing parameter ranges.
+
+    :param weights: mode e.g region default none
+    :output: 2d plot
+    """
+    conn = connectivity.Connectivity(load_default=True)
+    conn.configure()
+    conn.weights = weights
+    plot_connectivity(conn, num="tract_mode", plot_tracts=False)
+
+def run_sim(conn, cs, D, cv=3.0, dt=0.5, simlen=1e3):
```

Comment:
This is not about plotting. Why do you have such a function?

Reply:
removed